packages/core/src/utils/Timer.ts

```diff
@@ -15,10 +15,10 @@ const STOP_LABEL = '__stop';
  * All timestamps/durations returned are in milliseconds.
  */
 export default class Timer {
-    private timeProvider: TimeProvider;
+    private timeProvider: DefaultTimeProvider;
     private times: Record<string, Timestamp> = {};
 
-    constructor(timeProvider: TimeProvider = new TimeProvider()) {
+    constructor(timeProvider: DefaultTimeProvider = new DefaultTimeProvider()) {
```

Comment:
shouldn't it be `TimeProvider` type here and `DefaultTimeProvider` as default value?

```suggestion
    constructor(timeProvider: TimeProvider = new DefaultTimeProvider()) {
```

Reply:
Of course it should, nice catch!

Reply:
[Solved](https://github.com/DataDog/dd-sdk-reactnative/compare/88c6563310ef6b04ad5009ae0c2444edcc810272..1ed72d363d95857bc4868bfa4df620b8e9a9fb63)